pwiz_tools/Skyline/TestFunctional/RunToRunRegressionTest.cs

```diff
@@ -33,7 +33,6 @@ public class RunToRunRegressionTest : AbstractFunctionalTest
     {
         private const int REPLICATES = 5;
 
-        [TestMethod, NoParallelTesting]

```

Comment:
Looks like this is removing a test. Why?

Reply:
Error, thanks for catching it! Only meant to remove the parallel test prohibition.